provider/ansible.rb

```diff
@@ -235,66 +234,76 @@ def get_example(cfg_file)
       end
 
       def generate_resource(data)
-        # TODO: Azure switch
-        azure_generate_resource(data)
-
-        # target_folder = data.output_folder
-        # name = module_name(data.object)
-        # path = File.join(target_folder,
-        #                  "lib/ansible/modules/cloud/google/#{name}.py")
-        # data.generate(
-        #   data.object.template || 'templates/ansible/resource.erb',
-        #   path,
-        #   self
-        # )
+        # Multi Cloud Swith
+        if $cloud == 'MAC'
+          azure_generate_resource data
```

Comment:
Can you try whether the following code works? So we can minimize the changes to only one line.

```suggestion
        return azure_generate_resource(data) if is_azure_backend
```

Reply:
It works, and the code is modified in this way.